### DISCUSSION TITLE
개발하면서 어려웠던 점이 있으신가요?

### DISCUSSION THREAD

**Discussion by m04j00** [2022-IT-SHOW-Shade/android#57] [Q&A] [answered]
성공적인 프로젝트를 하셨는데, 개발하면서 어려웠던 점도 있으셨나요?

**Comment by 0pyaq0** [comment, marked as answer]
아무래도 디자인 전공이 아니다보니 효율적인 뎁스까지 고려하며 디자인을 구성하는 과정이 가장 어려웠던 것 같습니다.
그 밖에 나이스 API 사용 예제가 많이 없다보니 팀원이 꽤 헤맨 경험도 있었네요 😂
